src/main/java/com/dev/dao/impl/UserDaoImpl.java

```diff
@@ -48,4 +47,14 @@ public void add(User user) {
             throw new RuntimeException("Can't get all Users", e);
         }
     }
+
+    @Override
+    public User getById(Long id) {
+        try (Session session = sessionFactory.openSession()) {
+            User user = session.get(User.class, id);
+            return user;
```

Comment:
don't create this variable, simply return

Reply:
done